### PR TITLE
CADL of DPG as Resiliency tests

### PR DESCRIPTION
**Pull request by lmazuel**

### CODE REVIEW

File: packages/cadl-ranch-specs/http/resiliency/customization.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { app, json } from "../../api";
```
Comment (lmazuel) [inline packages/cadl-ranch-specs/http/resiliency/customization.ts:1]: @timotheeguerin probably this won't work now

File: packages/cadl-ranch-specs/http/resiliency/cadl-project.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+emitters:
```
Comment (lmazuel) [inline packages/cadl-ranch-specs/http/resiliency/cadl-project.yaml:1]: Should I commit this file?

File: packages/cadl-ranch-specs/http/resiliency/customization.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+app.category("dpg", () => {
```
Comment (lmazuel) [inline packages/cadl-ranch-specs/http/resiliency/customization.ts:3]: I will change the category to "azure"

File: packages/cadl-ranch-specs/http/resiliency/cadl-project.yaml (hunk 2 above)
Comment (timotheeguerin) [reply packages/cadl-ranch-specs/http/resiliency/cadl-project.yaml:1]: I think we should only need one `cadl-project.yaml` at the root if we want to configure the default emitter. SHould it be `cadl-autorest` though for now at least?

File: packages/cadl-ranch-specs/http/resiliency/package.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (timotheeguerin) [inline packages/cadl-ranch-specs/http/resiliency/package.json:1]: shouldn't need that file, the one at the root of `cadl-ranch-specs` should cover all the scenarios

**Review by timotheeguerin — APPROVED** [review]
Looks good, I think there is some inconsitentcy with the scenario name in the mock apis but might need to resolve that when the generated type is here

**Pull request merged**